packages/shorebird_cli/lib/src/commands/account/create_account_command.dart

```diff
@@ -53,11 +53,11 @@ To subscribe, run ${lightCyan.wrap('shorebird account subscribe')}.
 🔑 Credentials are stored in ${lightCyan.wrap(auth.credentialsFilePath)}.
 🚪 To logout, use: "${lightCyan.wrap('shorebird logout')}".
 
-A subscription is required to use Shorebird.
-To subscribe, run ${lightCyan.wrap('shorebird account subscribe')}.
+Your current plan is ${lightCyan.wrap('Hobby')}.
```

Comment:
We'll (eventually) want to wordsmith to talk about what this does and does not include.